cypress/integration/sign_in.spec.js

```diff
@@ -86,6 +88,131 @@ describe('Sign in flow', () => {
     });
   });
 
+  context.skip('Signing in - Okta (Skipped while feature disabled)', () => {
```

Comment:
Is there a way to dynamically set the feature flag so this test can run?

Reply:
Not at the moment. We could rely on cookies or something in the future? But that's extra work outside this scope.